ete3/tools/ete_build_lib/utils.py

```diff
@@ -651,4 +651,4 @@ def cmp(x, y):
     """cmp() exists in Python 2 but was removed in Python 3.
     This implements the same behavior on both versions.
     """
-    return (x > y) - (x < y)
+    return (x > y) ^ (x < y)
```

Comment:
??? this was working before. Why the change? the impact on other functions is high!